autobuild/startup_blockcreator.toml

```diff
@@ -0,0 +1,45 @@
+[startup.tfchaind]
+name = "core.system"
+
+
+[startup.tfchaind.args[startup.tfchaind]
+name = "core.system"
+
+
+[startup.tfchaind.args]
+name = "/bin/tfchaind"
+# should use default values here?
+args = ["--rpc-addr={TFCHAIND_RPC_ADDR}", "--api-addr={TFCHAIND_APIADDR}", "--persistent-directory", "/mnt/data " , "--network={TFCHAIND_NETWORK}"]
```

Comment:
Add the `--ethvalidation` flag, so field nodes by default run the ethereum validation